nd4j/nd4j-tools/src/main/java/org/nd4j/cli/TFtoFlatFileConverter.java

```diff
@@ -0,0 +1,148 @@
+package org.nd4j.cli;
+
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.nd4j.autodiff.samediff.SDVariable;
+import org.nd4j.autodiff.samediff.SameDiff;
+import org.nd4j.autodiff.samediff.transform.*;
+import org.nd4j.imports.graphmapper.tf.TFGraphMapper;
+import org.nd4j.imports.tensorflow.TFImportOverride;
+import org.nd4j.imports.tensorflow.TFOpImportFilter;
+import org.nd4j.linalg.api.buffer.DataType;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.*;
+
+/**
+ * Created by Yves Quemener on 12/7/18.
+ */
+public class TFtoFlatFileConverter {
```

Comment:
Need at least basic javadoc - on class and both methods.

Reply:
Done